.github/workflows/ci.yml

```diff
@@ -21,44 +22,47 @@ jobs:
           - '5.3.*'
           - '5.4.*'
           - '6.0.*'
-          - '6.1.*'
```

Comment:
Lowest support is 6.0 for 6.x, why skip 6.1 and 6.2?

As time passed by also consider to add 6.4.

Reply:
Can we bump minimum supported versions to `^5.4 || ^6.3 || ^7.0` as LTS Version, and latest actual version?